config-overrides.js

```diff
@@ -0,0 +1,29 @@
+const webpack = require("webpack");
+
+module.exports = function override(config) {
+  config.resolve.fallback = {
+    // url: require.resolve('url'),
+    // fs: require.resolve('fs'),
+    // assert: require.resolve('assert'),
+    // crypto: require.resolve('crypto-browserify'),
+    // http: require.resolve('stream-http'),
+    // https: require.resolve('https-browserify'),
+    // os: require.resolve('os-browserify/browser'),
+    buffer: require.resolve("buffer"),
+    stream: require.resolve("stream-browserify"),
```

Comment:
The latest Webpack (5+) no longer includes Node polyfills by default, so we have to add them here. We'll need to do this in Alberto and Gwen eventually.